client/Components/CodeEditor/CodeEditor.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import Select from '@material-ui/core/Select'
+import OutlinedInput from '@material-ui/core/OutlinedInput'
+import MenuItem from '@material-ui/core/MenuItem'
+import configEditor from './EditorConfigFunction'
+
+class AceEditor extends Component {
+  constructor() {
+    super()
+    this.state = {
+      code: '',
+      codeResponse: '',
+      codeError: '',
+      editor: {},
+      editorTheme: 'monokai'
+    }
+    this.configEditorBinded = configEditor.bind(this)
+  }
+
+  componentDidMount() {
```

Comment:
Just so I understand! So here:
1. We set the state of the editor field with ace.edit which appends an ace editor to the DOM
2. When the state is set, we callback with a fn that grabs the editor on the state, returns the current session being used and assigns it to editorSession variable
3. I'm looking for configEditorBinded method in the code but can't find it?

Reply:
Nevermind i see the configEditor file now

Reply:
You got it!